docs/examples/move/nft_marketplace/README.md

```diff
@@ -0,0 +1,179 @@
+The `rental_extension.move` is an extension adds functionality to enable item rentals.
+
+## Steps to Use the Marketplace
+
+### 1. Connect to the Network
+
+Connect to the IOTA network (e.g., using a faucet to obtain tokens).
+
+### 2. Install Kiosk
+
+By installation, we mean creating a Kiosk object and an OwnerCap, then transferring them to the caller.
+Run the following command to install the Kiosk module:
+
+```bash
+iota client call \
+    --package 0x2 \
+    --module kiosk \
+    --function default
+```
+
+### 3. Publish `nft_marketplace` package
+
+#### 3.1(Optional) Publish Kiosk rules modules if these are not present in the network you are using
+
+Publish Kiosk rules modules(package):
+
+```bash
+iota client publish iota/kiosk
+```
+
+After publishing, export the following variable:
+
+- `RULES_PACKAGE_ID`: The ID of rules package.
+
+#### 3.2 Publish marketplace extension
+
+Publish the marketplace_extension.move module:
+
+```bash
+iota client publish iota/docs/examples/move/nft_marketplace
+```
+
+After publishing, export the following variables:
+
+- `MARKETPLACE_PACKAGE_ID`: The ID of whole marketplace package.
+- `MARKETPLACE_PUBLISHER_ID`: The ID of the publisher object created during marketplace package publishing."
+
+### 4. Create an Clothing Store Item
+
+Create an item, for instance:
+
+```bash
+iota client call \
+    --package $MARKETPLACE_PACKAGE_ID \
+    --module clothing_store \
+    --function new_jeans
+```
+
+After creation, export the following variable:
+
+- `CLOTHING_STORE_ITEM_ID`: The ID of the published item (in this case, Jeans).
+
+### 5. Create a Transfer Policy
+
+`TransferPolicy` is a generic shared object acting as a central authority enforcing everyone to check their purchase is valid against the defined policy before the purchased item is transferred to the buyers. Object is specified by concrete type.
+`default` function creates `TransferPolicy` object and an OwnerCap, then transferring them to the caller.
+
+Set up a transfer policy for the created item using the command:
+
+```bash
+iota client call \
+    --package 0x2 \
+    --module transfer_policy \
+    --function default \
+    --gas-budget 10000000 \
+    --args $MARKETPLACE_PUBLISHER_ID \
+    --type-args "$MARKETPLACE_PACKAGE_ID::clothing_store::Jeans"
+```
+
+After publishing, export the following variables:
+
+- `ITEM_TRANS_POLICY`: The ID of the item transfer policy object.
+- `ITEM_TRANS_POLICY_CAP`: The ID of the item transfer policy object owner capability"
+
+### 6. Install the Extension on the Kiosk
+
+The install function enables installation of the Marketplace extension in a kiosk.
+Under the hood it invokes `kiosk_extension::add` that adds extension to the Kiosk via dynamic field.
+Install the marketplace extension on the created kiosk using the command:
+
+```bash
+iota client call \
+    --package $MARKETPLACE_PACKAGE_ID \
+    --module marketplace_extension \
+    --function install \
+    --args $KIOSK_ID $KIOSK_CAP_ID
```

Comment:
From where the `KIOSK_ID` and `KIOSK_CAP_ID ` ids are taken?
They were not mentioned previously in the document.
Should we add the information about this?

Reply:
Added at 2 point(`2. Install Kiosk`)